processor/k8sattributesprocessor/internal/kube/client.go

```diff
@@ -131,6 +131,19 @@ func New(logger *zap.Logger, apiCfg k8sconfig.APIConfig, rules ExtractionRules,
 			newReplicaSetInformer = newReplicaSetSharedInformer
 		}
 		c.replicasetInformer = newReplicaSetInformer(c.kc, c.Filters.Namespace)
+		err = c.replicasetInformer.SetTransform(
```

Comment:
We override `err` set by `c.informer.SetTransform`, which is expected to be returned as is. We can have another var here if we want to ignore this error. But I'm not sure why do we ignore this one and fail for the pod informer. I believe we need consistent behavior: ignore both with a warning or fail on any of them.